package.json

```
@@ -26,7 +26,7 @@
     "optimize:svg": "find assets -type f -name '*.svg' -print0 | xargs -0 -n 1 -P 6 svgo --multipass --pretty --enable=removeTitle",
     "optimize:png": "find assets -type f -name '*.png' -print0 | xargs -0 -n 1 -P 6 optipng",
     "watch:css": "chokidar 'components/**/*.scss' 'pages/**/*.scss' 'scss/**/*.scss' -c 'npm run build-client:scss'",
-    "watch:js": "chokidar 'app.jsx' 'main.jsx' 'js/**/*.js' 'components/**/*.jsx' 'pages/**/*.jsx' 'pages/**/*.js' 'webpack.config.js' -c 'npm run build-client:js'",
+    "watch:js": "chokidar 'app.jsx' 'main.jsx' 'js/**/*.js' 'components/**/*.jsx' 'components/**/*.js' 'pages/**/*.jsx' 'pages/**/*.js' 'webpack.config.js' -c 'npm run build-client:js'",
```

globs should understand `?` as "optional", so the following _should_ work?
```suggestion
    "watch:js": "chokidar 'app.jsx' 'main.jsx' 'js/**/*.js' 'components/**/*.jsx?' 'pages/**/*.jsx?  'webpack.config.js' -c 'npm run build-client:js'",
```

---

```'components/**/*.js?(x)'``` worked!